plugins/node/opentelemetry-instrumentation-express/src/utils.ts

```diff
@@ -145,3 +145,27 @@ export const asErrorAndMessage = (
   error instanceof Error
     ? [error, error.message]
     : [String(error), String(error)];
+
+/**
+ * Extracts the layer path from the route arguments
+ *
+ * @param args - Arguments of the route
+ * @returns The layer path
+ */
+export const getLayerPath = (args: unknown[]): string | undefined => {
+  if (args[0] instanceof RegExp) {
+    return args[0].toString();
+  }
+
+  if (typeof args[0] === 'string') {
+    return args[0];
+  }
+
+  if (Array.isArray(args[0])) {
+    return args[0]
+      .map(arg => (typeof arg === 'string' || arg instanceof RegExp ? arg : ''))
```

Comment:
Is there any situation where an array entry might be a type other than string or array? If so, it looks like this will result in an empty entry: `getLayerPath(['/test', 6, /test/]) === '/test,,/test/`

Reply:
Yes, looks like it's possible, I've added support for numbers.